packages/dina-ui/components/storage/StorageUnitBreadCrumb.tsx

```diff
@@ -37,7 +39,11 @@ export function StorageUnitBreadCrumb({
       {!hideThisUnit && (
         <li className="breadcrumb-item">
           <strong>
-            {storageUnit.id && !disableLastLink ? (
+            {storageUnit.id && !disableLastLink && newTab ? (
+              <Link href={`/collection/storage-unit/view?id=${storageUnit.id}`}>
+                <a target="_blank">{unitDisplayName}</a>
+              </Link>
+            ) : storageUnit.id && !disableLastLink ? (
```

Comment:
This can be simplified to go directly on the attribute:

```tsx
<Link href={`/collection/storage-unit/view?id=${storageUnit.id}`}>
  <a target={newTab ? "_blank" : false}>{unitDisplayName}</a>
</Link>
```

I believe false will not render the target attribute at all.

Reply:
refactored and tested at commit https://github.com/AAFC-BICoE/dina-ui/pull/676/commits/17d8761cbe40c748ad530c1c67d68484b88d2344